### ISSUE TITLE
[Talent/Paladin] Sanctified Wrath bypasses resilience

### ISSUE THREAD

**Issue by Oganku** [WoWManiaUK/Redemption#3323]
**Links:** http://www.wow-mania.com/armory/?spell=53376
                http://www.wow-mania.com/armory/?spell=31884

**What is Happening:** The talent [Sanctified Wrath](http://www.wow-mania.com/armory/?spell=53376) seems to be bypassing resilience and/or bypassing more than 50% damage reduction effects on players. I unfortunately do not have concrete data, but I would like if someone could investigate as the damage it allows seems way too high.

**What Should happen:** I don't think paladins should be able to kill players with 1400+ resilience in 2 or 3 seconds while under the effect of [Avenging Wrath](http://www.wow-mania.com/armory/?spell=31884).

**Comment by Dranah** [comment]
I've literally tested it 1hour ago on another report https://github.com/WoWManiaUK/Redemption/issues/3321
The talent works as intended... And i'll repeat once more, **it allows to bypass resilience as well.** 
Why does it feel like the "spell does full damage not half"?
Because it doesn't calculate from the "damage you normally do". Let me illustrate:

Abilities have a base damage calculated from player stats before mitigation of target. Say your ability hits for 3k on a player, but that's already past resi, armor, whatever talents and buffs that reduce his damage taken. In most cases the player has at least 30% resi + the other effects, so it's safe to assume atleast 40% mitigation.
This means the spell would actually hit for 5k without any mitigation. That is the actual base damage to calculate from.
The talent Sanctified Wrath grabs this fully unmitigated damage and allows 50% of it to simply hit your enemy ignoring every defense he has. In this example, this would be around 2,5k damage. That means this 50% component alone will hit for almost as much as the full damage did before.
The other 50% undergoes mitigation as before and if the enemy has popped a defensive cooldown it will get reduced further, so for example on a dispersed priest the other 2,5k will do maybe 200 damage only.
If you add up the two, you still land 2,7k damage on him, which is nearly as much as you did before he popped disperse and you popped AW.

I

**Comment by Dranah** [comment]
I will leave this open to allow other testers to take a look but I can't find any problem with the spell.
Yes it bypasses resilience too, because guess what, resi is also a damage reduction effect. 
Nothing is broken here.

**Comment by Sudrynox** [comment]
Dranah how nothing is broken when pala rets hit 6k under dmg reduction .... I play wow wotlk since 2007 I've never seen pala rets hit like that ..

**Comment by Dranah** [comment]
Well, the maths checks out. It's a different story that some players claim it should not affect resilience. But I can't find any evidence for that.
As I said if someone links a reliable source that proves that, we can change it.

**Comment by Sudrynox** [comment]
impossible to kill a shadow priest in Dispersion like that? I already tested on my sp against a pala ret I was 10k hp I did Dispersion he made wings he killed me under Dispers lol

**Comment by Oganku** [comment]
Could it be that instead of having 50% of the damage do normal damage and the other 50% to do full damage (ignoring reductions), it basically reduces damage reduction by 50% for all damage? Just a thought I had.

**Comment by Dranah** [comment]
Nope, the calculation is done right. The only questionable part if bypassing resi is intended or not.
As I said that needs evidence backup.

**Comment by Sudrynox** [comment]
I'm sure that pala ret when he pop wings he is able to remove the resilience and all dmg reduction at 100%

**Comment by Dranah** [comment]
@Sudrynox if you cannot contribute any actual useful information, please stay silent

**Comment by Sudrynox** [comment]
is not my job testing bugs , you should do it  cya dude 👍

**Comment by Kitzunu** [comment]
I made a little research, I couldn't find much but I found a post from 2010 on a WoW Emulator site where they state:

>Seegos Posted June 5, 2010
>>darkstalker said:
>>>resilience not supposed to be ignored in any way afaik (there was a blizz post about it somewhere)

>Maybe you're talking about this:

>>Ghostcrawler said:
>>>I edited my previous post to correct my mistake. **Resilience is not treated as a damage reduction effect.** Nevertheless, the team still felt like the 3.1.3 change to Chaos Bolt was appropriate. You can find more details above or in the related Chaos Bolt thread.

The source of the comment by Ghostcrawler is http://forums.worldofwarcraft.com/thread.html?topicId=17367979734&pageNo=14#272 however the post is removed since Blizzard decided to get a new Forum for World of Warcraft a while ago...
Back in the day, Ghostcrawler was one of the Lead Developers for blizzard.

Either way, after reading that quote to me it sounds like resilience should **not** be reduced by Sanctified Wrath.

You can also read in this post https://www.engadget.com/2008/07/19/wrath-beta-patch-notes-paladin-part-v-the-retribution-tree/?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAJ1MAkrry28GA4qXD3Rno4qc9uqnYvlUgN_3gslBwpMcslUo4qG6P5NfNeEzdc_zvDik1W9PCWY_CP7GPc0rsbSW5kY87JGNTWEen2LhvvVN6cJxiB2IUDzeUg8nAs1bKGksyMbl47vjbWxb69Pk5ccrioRk1fGZqPmfQ7M22WSP
>The best part of this talent -- for me, at least -- is the bypass in damage reduction effects. Armor and Resilience... ignored. 50% of all damage done for 20 seconds ignoring damage reduction!

Which then sounds like it is ignoring Resilience AND Armor, though these were from Beta wrath.

in this forum post, you have a guy saying he dumped Sanctified Wrath as it got nerfed, and to me, I feel like, why would you dump Sanctified Wrath if it was reducing Resil/Armor
https://www.wowhead.com/forums&topic=50958#p506662

To me, it is weighing towards that this issue is actually valid. But it is so long ago now so most websites with proof from this time are gone, sadly.

**Comment by Dranah** [comment]
Well the latter explains what got nerfed about it, hammer of wrath not being so powerful anymore, and forbearance being added. I do feel he would have mentioned if there was any other change as not affecting resilience would have been a significant one.
So if we put together the picture, it was affecting Resilience in the beta and certainly up to 3.1.
It might have received another change after, but there is zero proof of that still.
I could find the chaos bolt changes in the patchlist but nothing about Sanctified Wrath.
For me this is not enough to nerf paladins to the ground.

**Comment by Dranah** [comment]
Ok, after a ton of digging I may have found something that can confirm Kitzu's thoughts of above.
Blue post discussion on a not-yet-erased forum:
https://blue.mmo-champion.com/topic/5607-proof-that-chaosbolt-doesnt-ignore-resilience/
While the topic is on chaos bolt, it is relevant and generalized at this point:
"There was a rumor, apparently long-standing because I believed it, that things that ignored damage reduction effects also ignored resilience.
When the designers made the change to Chaos Bolt for 3.1.3, they verified that this was NOT the case."

So basically chaos bolt used to ignore damage reduction effects pre 3.1 and while it was hotfixed (because the tooltip never stated this), a ton of testing had been done and confirmed that this did not include resilience in the first place. Resilience is not considered a damage reduction effect per se, therefore Sanctified Wrath should not affect it either.

This needs fixing.

**Comment by Dranah** [comment]
As an additional thought i'll add if you read the entire conversation on that link, Blizzard staff themselves admitted they had no idea how to classify resilience until 3.1 and as such, Ghostcrawler had been posting controversial informations regarding it, prior to somebody actually making a numeric test to prove it.
He confessed his mistake and apologized, but got a giant shitstorm on his head nonetheless. 
It's worth a read, i laughed my ass off.

**Comment by Oganku** [comment]
Thanks a lot to everyone who contributed to digging up evidence on this obscure topic. Cheers to Dranah and Kitzunu! 👍

**Comment by Keader** [comment]
Checking in code, only resistences and armor is ignored, resilience is still applied.

**Comment by Oganku** [comment]
Excuse me, but how can this not be reproduced. Just go in the game with 1400+ resilience and let a paladin hit you with wings. You die in 3 gcd's at most. I guess the code looking legit is the reason this hasn't been fixed yet but it definitely does not feel right whatsoever.

**Comment by Dranah** [comment]
I have just made a test.
Fix damage ability (judgement) vs rogue with 900 resi.
The damage difference between no wings and wings is exactly 20%, which is the boost from wings. If it caused any kind of bypass, it would be more than 20%, since on top of the wings boost, a part of the spell would do more damage than before.
However that's not the case.
So, i'm glad we discussed that it should, indeed, respect resi and the tests show that it does.
I've no idea why paladins are eating you in pvp, but this is not bugged.
